src/components/PassphraseForm.jsx

```diff
@@ -82,7 +82,6 @@ class PassphraseForm extends Component {
       newPassphraseTouched &&
       newPassphraseMatch &&
       strength.label !== 'weak' &&
-      hint &&
```

Comment:
the commit prefix that respects the convention is lowercase.

feat: Password...

Reply:
btw "optional" with one "n"